org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  curBucket:
+    Type: String
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub ""
+      ReportVersioning: False
+      TimeUnit: "hourly"
+      Compression: "parquet"
+      Format: "parquet"
+      S3Region: !Ref primaryRegion
+      S3Prefix: "hourly"
```

Comment:
consider parameterizing these configs so that you can create CUR with different parameters using the same template?

Reply:
I moved the frequency to a parameter, but left the compression and format as opinionated values. Let me know if they should be parameterized as well.